it/graphql-multipart/build.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+    implementation project(':graphql')
+    testImplementation libs.spring.boot2.starter.web
+    testImplementation libs.spring.boot2.starter.test
```

Comment:
Is Spring Boot dependency needed? Only Spring Framework API looks used.

Reply:
I used it because I use [MockMultipartFile](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/mock/web/MockMultipartFile.html).

Reply:
I see.